assets/externalized/strategic-map-npc-placements.json

```diff
@@ -1,9 +1,11 @@
 /**
- * Lists the sectors where randomly placed NPCs can appear. Each NPC here has his own corresponding code sections.
+ * Lists the sectors where randomly placed NPCs can appear. 
+ * 
+ * Adding other NPCs here will not have any effect, as each NPC here requires his own corresponding sections in codebase.
  *
  * Fields:
  *  - profileId:    The numeric ID of the NPC merc profile.
- *  - sectors:    Sectors in which the NPC can appear. All of these sectors should to have NPC merc+-+ placed, or else the NPC may get "assigned" to sector but not appearing.
+ *  - sectors:    Sectors in which the NPC can appear. Must not be empty. All of these sectors should to have NPC merc placed, or else the NPC may get "assigned" to sector but not appearing.
```

Comment:
Is there some deeper engine need for this? Because now you can't for example disable Hamous ever appearing just by editing the sectors out.

Reply:
No, I don't think so. 

Indeed we can support disabling NPCs by adding empty check and allowing empty sector lists instead.